dap_taltech/utils/data_getters.py

```diff
@@ -0,0 +1,67 @@
+"""
+This script contains:
+    A class to load data from either a local directory or s3 data that is relevant
+    across tutorials.
+    The methods in the class have detailed docstrings that describe the data.
+"""
+import os
+import pandas as pd
+
+from dap_taltech import (PROJECT_DIR,
+                         PUBLIC_DATA_FOLDER_NAME,
+                         BUCKET_NAME,
+                         logger)
+
+class DataGetter(object):
+    """Class to load datasets relevant across different tutorials for TalTech HackWeek 2023.
+
+    Parameters
+    ----------
+    verbose : bool, optional
+        Set the logger level. If verbose, set logger level to INFO. Else, set logger level to ERROR.
+        Defaults to True.
+    local : bool, optional
+        If True, load data from local data directory. Else, download data from open dap-taltech s3 bucket.
+
+    """
+
+    def __init__(
+        self,
+        verbose=True,
+        local=True
+    ):
+        self.verbose = verbose,
+        self.local = local
+        if self.verbose:
+            logger.setLevel('INFO')
+        else:
+            logger.setLevel('ERROR')
+        if self.local:
+            self.data_dir = os.path.join(PROJECT_DIR, PUBLIC_DATA_FOLDER_NAME)
+            logger.info(f'Loading data from {self.data_dir}/')
+            if not os.path.exists(self.data_dir):
+                logger.warning(
+                    "Neccessary data files are not downloaded. Downloading neccessary files..."
+                )
+                os.system(f'aws s3 sync s3://{BUCKET_NAME}/data {self.data_dir}')
```

Comment:
Interesting to use cmd calls. Isn't the risk to have environment-specific issues pop up here? Isn't it easier to use Python native libraries instead (boto3)